home.php

```diff
@@ -71,4 +71,4 @@
         }
     }
     ?>
-</table>
\ No newline at end of file
+</table>
```

Comment:
Používat proměnné :+1: 

```
 if (isset($_SESSION["unauthorized_entry"])) {
        echo htmlspecialchars($_SESSION["unauthorized_entry"]);
        unset($_SESSION["unauthorized_entry"]);
    }
```

Vykreslit data jde jednodušeji..

- nejlépe bych si data vyžádal předem (mimo teplate) a uložil do pole:
`$dog_info = array($dog_name, $dog_breed, $dog_age, $dog_origin, $dog_description, $dog_owner);`
- pak bych vykreslil všechny sloupce:
```
<tr>
foreach ($dog_info as $value) {
   echo "<td> . $value . </td>"
}
</tr>
```
(toto je příklad nejjednoduššího zlepšení/zjednodušení, šlo by to ještě líp tak, že si předem fetchneš data psů a pak pro každý záznam vytvořít nový řádek v tabulce :+1: )

Reply:
Co prosím znamená, že si můžu předem "fetchnout" data psů?

Reply:
https://dictionary.cambridge.org/us/dictionary/english/fetch?q=to+fetch

jde o získání dat :slightly_smiling_face: